test/integ_tests/mock_utils.py

```diff
@@ -193,7 +193,7 @@ class Boto3SessionAllWrapper(SessionWrapper):
     def __init__(self):
         super().__init__()
         boto3.Session = self
-        self._default_region = "us-west-2"
+        self._default_region = "us-west-1"
```

Comment:
Same question as in the other file, is it that ankaa-2 and ankaa-3 are in different regions?

Reply:
We should look to leave this default region as typically this is what we assume for other tests (SDK integs, etc)

Reply:
I managed to turn it back to "us-west-2" but editing L16 in `test/integ_tests/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job_mocks.py`